com/bl/ttt/main.py

```diff
@@ -1,21 +1,88 @@
 from com.bl.ttt.tictactoe import Tictactoe, TictactoeMove
+from random import randrange
+import time
 
 game_over = False
 player_tries_again = True
 get_next_move = True
 
 
-def process_another_move(game):
+def get_game_start():
+    while True:
+        try:
+            new_game_input = input("Would you like to start a game of Tic Tac Toe? Please enter 'y' or 'n': ").upper()
+            if new_game_input is not None and (new_game_input == 'N' or new_game_input == 'Y'):
+                return new_game_input
+            else:
+                print('That was an invalid input: please try again.')
+                continue
+        except (Exception, ValueError):
+            break
+
+
+def get_game_type():
+    while True:
+        try:
+            game_type_input = input("What kind of game will you play: enter 'h' for human or 'c' for computer: ").upper()
+            if game_type_input is not None and (game_type_input == 'H' or game_type_input == 'C'):
+                return game_type_input
+            else:
+                print('That was an invalid input: please try again.')
+                continue
+        except (Exception, ValueError):
+            break
+
+
+def get_first_move(game):
     player_id, cell = input('Please enter player X or O, a space, and then a cell from 0 to 8: ').split(' ')
+    game.player1 = player_id
+    if game.player1 == 'X':
+        game.player2 = 'O'
+    else:
+        game.player2 = 'X'
```

Comment:
Assigning things in another class directly - also pulling things without getters